### PR TITLE
Answer questions ons JQuery and the DOM

### PR DESCRIPTION
**Pull request by susandaiute**
Largely unfinished due to the MBTA.

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -3,28 +3,24 @@
...
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$li.parentNode.form
```
Comment (RDegnen) [inline diagnostic.md:16]: `$('ol').children()`

File: diagnostic.md [hunk 2]
```diff
@@ -3,28 +3,24 @@
...
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$(.listerItem).parentNode.lastChild;
```
Comment (RDegnen) [inline diagnostic.md:23]: `$('li').parent().siblings('form')`

**Comment by RDegnen** [comment]
2 of 8, its noted that you didn't have as much time to complete the diagnostic

**Comment by susandaiute** [comment]
Hi Ross, I was here at 9:12.  Is that considered late?

Thanks for the notice!

**Comment by RDegnen** [comment]
Sorry I worded that wrong, I updated my comment. 9:12 is fine

**Comment by susandaiute** [comment]
Great! Thank you

**Pull request closed without merging**